### PR TITLE
getPlatformString

### PR DESCRIPTION
**Pull request by dimitre**
This PR adds the special function getPlatformString for PG
it returns the actual development platform you are using. you can develop on macOS but not on iOS, you will be able to run project generator in Raspberry PI but not on Android.

### CODE REVIEW

**Comment by danoli3** [comment]
This is good! I think we should allow this setting in advanced options too directly

File: commandLine/src/projects/baseProject.h [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#define PG_VERSION "41"
+#define PG_VERSION "42"
```
Comment (danoli3) [inline commandLine/src/projects/baseProject.h:3]: this gonna conflict

File: commandLine/src/utils/Utils.cpp [hunk 2]
```diff
@@ -36,6 +36,57 @@
 
 using std::unique_ptr;
 
+std::string exec(const char* cmd) {
```
Comment (danoli3) [inline commandLine/src/utils/Utils.cpp:39]: can we call this execute_popen

**Review by danoli3 — CHANGES_REQUESTED** [review]
And rebase and stuff and lets get this in

File: commandLine/src/utils/Utils.cpp (hunk 2 above)
Comment (dimitre) [reply commandLine/src/utils/Utils.cpp]: sure thing

**Comment by dimitre** [comment]
@danoli3 ok all updated and compiling.

**Comment by danoli3** [comment]
oh wait what happened to linux and windows,

**Comment by danoli3** [comment]
shall merge tomorrow

**Comment by dimitre** [comment]
merge master, fix conflicts, version bump. good to go @danoli3

**Pull request merged**